sdks/common/fn-api/src/main/proto/beam_fn_api.proto

```diff
@@ -277,6 +278,7 @@ message Metrics {
 }
 
 message ProcessBundleProgressResponse {
+  // (Required)
```

Comment:
I would mention within the InstructionRequest, that for any unsupported request type, an error should be returned with a matching id and then mention on the ProcessBundleProgressRequest that this is an optional request to be handled and is used to support advanced SDK features such as SplittableDoFn.

Reply:
Done